EasyBusiness/server/database/db_build.sql

```diff
@@ -0,0 +1,46 @@
+BEGIN;
```

Comment:
Again, this database code doesn't relate to the title of the commit message "Login form design is completed". 

here are the slides from a great talk about git commits by Joel Chippindale https://www.slideshare.net/joelchippindale/simplify-writing-code-with-deliberate-commits-london-python-meetup

To summarise: make clean, atomic commits! (this will be more important during your client project when you're working a team of 4, it may seem like a small thing but it can make all the difference)